packages/@react-spectrum/toast/src/ToastContainer.tsx

```diff
@@ -47,7 +47,7 @@ function wrapInViewTransition<R>(fn: () => R): R {
       flushSync(() => {
         result = fn();
       });
-    });
+    }).ready.catch(() => {});
```

Comment:
bit of an odd question, not having much experience with view transitions yet
why do we call the flushSync above, and not in `then` on ready?

Reply:
I haven't seen it used that way, but I'll take a deeper look at the spec

Reply:
DOM updates must happen within the `startViewTransition` callback. Easiest way to do that is `flushSync` since React doesn't provide a promise that tells us when rendering is complete. I think we just need to catch errors in case the browser aborts the transition for some reason. In that case we don't want the whole page to crash.